tests/test_utils/test_activation_checkpointing.py

```diff
@@ -62,3 +61,54 @@ def test_activation_checkpointing(cpu_offload):
     # other tests will fail if running together with this test
     # as other tests can't overwrite the seed set by this test
     reset_seeds()
+
+
+@pytest.mark.gpu
+@pytest.mark.parametrize("cpu_offload", [True, False])
```

Comment:
If these two test functions only differ by the `use_reentrant` variable, you can just add `parameterize` here instead of creating a duplicated function.

Reply:
okay I will modify this